Godeps/_workspace/src/github.com/syncthing/protocol/message.go

```diff
@@ -28,6 +30,9 @@ func (f FileInfo) String() string {
 }
 
 func (f FileInfo) Size() (bytes int64) {
+	if f.ActualSize > 0 {
+		return f.ActualSize
+	}
 	if f.IsDeleted() || f.IsDirectory() {
 		return 128
```

Comment:
First time I noticed this magic constant. Might need a ref/comment why this is 128?